static_src/js/main.js

```diff
@@ -267,7 +267,25 @@ function initializeJS() {
 	})();
 })(jQuery);
 
+var swimWithTheMermaids = function() {
+
+  // Are there any diagrams here? if not skip.
+  if ( 1 > $( '#js-stage' ).length ) {
+    return undefined;
+  }
+
+  mermaid.mermaidAPI.render(
```

Comment:
Does this assume there's either zero diagrams or one? Since we're selecting by (hopefully unique) id, would this work with multiple diagrams?

Reply:
yeah, I think I'm going to have to pass in nodes rather than a string to the render function, for now I'm just copying over Scuttle, but I'm not sure how I'm gonna get the styles over. Probably going to need to publish it to get it over in `cg-style` as a `npm install` 🤷‍♂️